src/object/srv_obj.c

```diff
@@ -687,6 +687,133 @@ obj_echo_rw(crt_rpc_t *rpc, daos_iod_t *split_iods, uint64_t *split_offs)
 	orwo->orw_map_version = orw->orw_map_ver;
 }
 
+
+int
+ec_bulk_transfer(crt_rpc_t *rpc, crt_bulk_op_t bulk_op, bool bulk_bind,
```

Comment:
Where did this code come from?  I'm not expecting new functionality in this PR and I can't see where it's called or where the bulk of this code might have moved from.  It's not from a bad merge is it?

Reply:
It's come from PR #1169

Reply:
It looks bad merge. Will rebase on top of master...